src/main/java/nl/tudelft/contextproject/webinterface/WebServer.java

```diff
@@ -332,10 +293,8 @@ public static Cookie findCookie(String name, Cookie[] cookies) {
 	 * 		the newly created cookie
 	 */
 	public static Cookie createCookie(HttpServletRequest request) {
-		//Use the session to generate the Session ID.
```

Comment:
Good idea, this comment in and of itself doesn't explain that, which is why I removed it.